source/concepts/block.rst

```diff
@@ -59,6 +48,35 @@ Serialization of the block header.
 
    <a href="https://en.wikipedia.org/wiki/Radix_tree" target="_blank">patricia tree</a>
 
+********
+Creation
+********
+
+|codename| calls the first block in the chain the **nemesis block**.
+The first block is defined before launching a new network and sets the distribution of the network currency mosaics.
+
+The process of creating the subsequent new blocks is called :doc:`harvesting <harvesting>`.
+
+Blocks are created by :doc:`accounts <account>` based on their importance score.
```

Comment:
Blocks are created by :doc:`accounts <account>`, which are  chosen by the :doc:`consensus algorithm <consensus-algorithm>`based on their importance score.